docs/source/user-guide/configuration/configuration-settings.rst

```diff
@@ -0,0 +1,807 @@
+can disable the long test prefix. This is not recommended.
+
+.. code-block:: yaml
+
+   conda-build:
+       long_test_prefix: false
+
+The default is ``true``.
+
+.. _pypi-upload-settings:
+
+PyPI upload settings (conda-build 3.0+) (pypirc)
+------------------------------------------------
+
+Unset by default. If you have wheel outputs in your recipe, conda-build will
+try to upload them to the PyPI repository specified by the ``pypi_repository``
+setting using credentials from this file path.
+
+.. code-block:: yaml
+
+   conda-build:
+       pypirc: ~/.pypirc
+
+.. _pypi-repository:
+
+PyPI repository to upload to (conda-build 3.0+) (pypi_repository)
+-----------------------------------------------------------------
+
+Unset by default. If you have wheel outputs in your recipe, conda-build will
+try to upload them to this PyPI repository using credentials from the file
+specified by the ``pypirc`` setting.
+
+.. code-block:: yaml
+
+   conda-build:
+       pypi_repository: pypi
+Expansion of environment variables
+==================================
+
+Conda expands environment variables in a subset of configuration settings.
+These are:
+
+- ``channel``
+- ``channel_alias``
+- ``channels``
+- ``client_cert_key``
+- ``client_cert``
+- ``custom_channels``
+- ``custom_multichannels``
+- ``default_channels``
+- ``envs_dirs``
+- ``envs_path``
+- ``migrated_custom_channels``
+- ``pkgs_dirs``
+- ``proxy_servers``
+- ``verify_ssl``
+- ``allowlist_channels``
+
+This allows you to store the credentials of a private repository in an
+environment variable, like so:
+
+.. code-block:: yaml
+
+  channels:
+    - https://${USERNAME}:${PASSWORD}@my.private.conda.channel
+.. _threads:
+
+Configuring number of threads
+=============================
+
+You can use your ``.condarc`` file or environment variables to
+add configuration to control the number of threads. You may
+want to do this to tweak conda to better utilize your system.
+If you have a very fast SSD, you might increase the number
+of threads to shorten the time it takes for conda to create
+environments and install/remove packages.
+
+**repodata_threads**
+
+* Default number of threads: None
+* Threads used when downloading, parsing, and creating repodata
+  structures from repodata.json files. Multiple downloads from
+  different channels may occur simultaneously. This speeds up the
+  time it takes to start solving.
+
+**verify_threads**
+
+* Default number of threads: 1
+* Threads used when verifying the integrity of packages and files
+  to be installed in your environment. Defaults to 1, as using
+  multiple threads here can run into problems with slower hard
+  drives.
+
+**execute_threads**
+
+* Default number of threads: 1
+* Threads used to unlink, remove, link, or copy files into your
+  environment. Defaults to 1, as using multiple threads here can
+  run into problems with slower hard drives.
+
+**default_threads**
```

Comment:
```suggestion
**``repodata_threads``**

* Default number of threads: None
* Threads used when downloading, parsing, and creating repodata
  structures from ``repodata.json`` files. Multiple downloads from
  different channels may occur simultaneously. This speeds up the
  time it takes to start solving.

**``verify_threads``**

* Default number of threads: 1
* Threads used when verifying the integrity of packages and files
  to be installed in your environment. Defaults to 1, as using
  multiple threads here can run into problems with slower hard
  drives.

**``execute_threads``**

* Default number of threads: 1
* Threads used to unlink, remove, link, or copy files into your
  environment. Defaults to 1, as using multiple threads here can
  run into problems with slower hard drives.

**``default_threads``**
```

Reply:
Let's turn these into proper sub-headlines as in the other sections above, instead of just bold code text, that means a href is automatically created for them (which there isn't right now)